pandas/core/reshape/merge.py

```diff
@@ -952,6 +959,51 @@ def _validate_specification(self):
         if len(self.right_on) != len(self.left_on):
             raise ValueError("len(right_on) must equal len(left_on)")
 
+    def _validate(self):
+        # Get merging series:
```

Comment:
The alternate? Shortest safe one I have is:

    pd.concat(list(map(pd.Series, left_keys)), axis='columns')